observability/README.md

```diff
@@ -250,10 +252,7 @@ For this quickstart, a configuration has already been enabled for every service
 
 Note you did not introduce any dependency on Zipkin into the calculator app code or deployment yaml files. The Zipkin Dapr component is configured to read tracing events and write these to a tracing backend.
 
-Now deploy the distributed calculator application to your cluster following the instructions found in the [distributed-calculator](https://github.com/dapr/quickstarts/blob/master/distributed-calculator/README.md) quickstart. Then browse to the calculator UI.
-
-The quick version for starting the calculator quickstart:
-
+Now deploy the distributed calculator application to your cluster:
```

Comment:
Need \n after this line (The markdown parser I'm using doesn't correctly recognize HTML comments unless they have \n before and after)